helm-chart/openj9-jitserver-chart/README.md

```diff
@@ -0,0 +1,228 @@
+-->
+
+# OpenJ9 JITServer Helm Chart
+
+## Introduction.
+The OpenJ9 JITServer Helm Chart allows you to deploy, manage JITServer technology into Kubernetes or OpenShift clusters and make it possible for your Java applications to take advantage of it. JITServer is a technology that offers relief from the negative side-effects of the JIT compilation which are mostly felt in short lived applications running in resource constrained environments. The idea behind the technology is to decouple the JIT compiler from the JVM, containerize it and let it run as-a-service, in the cloud, where it can be managed intelligently by an of-the-shelf container orchestrator like k8s. JITServer is available as a "tech preview" in the OpenJ9 JVM and is supported on Linux on x86-64, Linux on Power and Linux on Z running Java8 or Java11
+
+## Getting started with OpenJ9 JITServer Technology.
+JITServer technology can be used for any Java application running on the latest release of OpenJ9 on supported Java versions and Linux platforms. You can make use of JITServer technology following the steps below:
+
+1. Install the JITServer using Helm Chart
+2. Enable your application to use JITServer
+
+### 1. Install the JITServer using Helm Chart.
+#### a. Verify the application JDK version and select the appropriate JITServer image.
+Important: The client JVM and the JITServer instance it connects to must use the same (compatible) version. The easiest way to ensure this is using the same release on both JITServer JDK and application JDK.
+
+Inside your Java application image, run `java -version` to check the JDK release version. In the example below, the `0.21.0` release of Java 11 JDK on `Linux amd64` architecture is used as shown in `build openj9-0.21.0, JRE 11 Linux amd64-64-Bit`. 
+```bash
+$ java -version
+openjdk version "11.0.8" 2020-07-14
+OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.8+10)
+Eclipse OpenJ9 VM AdoptOpenJDK (build openj9-0.21.0, JRE 11 Linux amd64-64-Bit Compressed References 20200715_697 (JIT enabled, AOT enabled)
+OpenJ9   - 34cf4c075
+OMR      - 113e54219
+JCL      - 95bb504fbb based on jdk-11.0.8+10)
+```
+
+Find the corresponding image from [AdoptOpenJDK Docker Hub](https://hub.docker.com/_/adoptopenjdk) through keywords search of architecture, Java version and release. In this example, `0.21.0` release of Java 11 is used in the application image, thus the corresponding image tag is [`11.0.8_10-jdk-openj9-0.21.0`](https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/11.0.8_10-jdk-openj9-0.21.0/images/sha256-996bb5bca1ae3fbab7bcd6ed4b19776eb1e176727024ce8866a75536137949b7?context=explore). The image tag will be used in the following step.
+
+#### b. Install the JITServer using the selected image and the Helm Chart.
+Add OpenJ9 JITServer helm chart repository to the Helm client.
+``` bash
+helm repo add openj9 https://<updatewhenweknowtheurl>stable/openj9/repo/index.yaml
```

Comment:
The initial idea was to use this github repo as the helm chart repo. That was the purpose of the tarball file in helm-chart/repo. If hosting that tarball in github is not a good idea we need to find a different place.

Reply:
git repos that we expect users to clone are not a great place to put binaries.  Especially if this will need to be updated for every release.

Maybe on artifactory?

Reply:
Looking into artifactory as an alternative option.